docs/source/user_guide/runtime-conf.md

```diff
@@ -215,6 +215,11 @@ The Apache Airflow API endpoint you want to utilize. This setting is required.
 
 Example: `https://your-airflow-webserver:port`
 
+##### Apache Airflow user namespace (user_namespace)
+The namespace used to run your DAG in Apache Airflow. The Kubernetes namespace must be configured with the correct permissions prior to use in Apache Airflow. This setting is Optional. 
```

Comment:
What's the default?

Reply:
Should we just make this a required field with a default instead of an optional field?

Reply:
We dont make it required in kfp schema, so I rescind my previous statement.

Reply:
Let me clarify: if no value is specified in the RTC the exported DAG contains this
```
notebook_op_696eb777_f24c_40be_875b_d16ed71d1254 = NotebookOp(
    name="a_notebook",
    namespace="default",
```

which indicates that an explicit default value of "default" is  applied somewhere. There are two issues:
- the default should be defined in the schema
- the default should be documented

Reply:
PR updated with changes